Assets/Mirror/Runtime/Transport/Websocket/Server.cs

```diff
@@ -131,7 +133,7 @@ async Task ProcessTcpClient(TcpClient tcpClient, CancellationToken token)
                 WebSocketHttpContext context = await webSocketServerFactory.ReadHttpHeaderFromStreamAsync(tcpClient, stream, token);
                 if (context.IsWebSocketRequest)
                 {
-                    WebSocketServerOptions options = new WebSocketServerOptions() { KeepAliveInterval = TimeSpan.FromSeconds(30), SubProtocol = "binary" };
+                    WebSocketServerOptions options = new WebSocketServerOptions() { KeepAliveInterval = KeepAliveTimer == 0 ? TimeSpan.Zero : TimeSpan.FromSeconds(KeepAliveTimer), SubProtocol = "binary" };
```

Comment:
could you do the TimeSpan part in the line above this one, and then pass the variable to KeepAliveInterval?
it would be a lot easier to read :)

Reply:
done